sys-fs/udev/udev-249-r2.ebuild

```diff
@@ -22,6 +22,9 @@ else
 	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86"
 fi
 
+SRC_URI+="
+	elibc_musl? ( https://dev.gentoo.org/~soap/distfiles/${P}-musl-patches.tar.xz )"
```

Comment:
Could you add a comment indicating where these patches came from?

Reply:
I wonder how different it to the tmpfiles patchset.
looking at tar archive yours it a bit smaller.
    # http://cgit.openembedded.org/openembedded-core/tree/meta/recipes-core/systemd/systemd
    # check SRC_URI_MUSL in systemd_${PV}.bb file for exact list of musl patches
^ I just extract those patches without much filtering.